ParselyTracker/HttpClient.swift

```diff
@@ -1,17 +1,16 @@
 import Foundation
 import os.log
 
-import SwiftHTTP
+import Alamofire
 
 class HttpClient {
     static func sendRequest(request: ParselyRequest) {
         os_log("Sending request to %s", log: OSLog.tracker, type: .debug, request.url)
-        HTTP.POST(request.url, parameters: request.params, headers:request.headers as? [String: String],
-                  requestSerializer: JSONParameterSerializer())
-        { response in
+        
+        Alamofire.request(request.url, method: .post, parameters: request.params, encoding: JSONEncoding.default, headers: request.headers as? HTTPHeaders).responseJSON {
```

Comment:
What are the implications of casting headers to `HTTPHeaders` as opposed to `String` here?

Reply:
`HTTPHeaders` is just a [custom type that's a dictionary `[String:String]`](https://github.com/Alamofire/Alamofire/blob/75bba56748359f297a83f620d45f72cf4ebee4e7/Source/Request.swift#L67).

I suppose I could define our `request.headers` as an instance of `HTTPHeaders` but it felt like that was too tight of a coupling. We just shouldn't change our `request.headers` to not be a `[String:String]` but I believe that would cause a compile time error?

Reply:
Thanks for the explanation, I don't see any changes necessary here given that.